src/reachy2_sdk/parts/mobile_base.py

```diff
@@ -173,6 +173,26 @@ def set_speed(self, x_vel: float, y_vel: float, rot_vel: float) -> None:
         )
         self._mobility_stub.SendDirection(req)
 
+    def translate_by(self, x: float, y: float, timeout: Optional[float] = None) -> None:
+        """Send a target position relative to the current position of the mobile base.
+
+        (x, y) define the translation wanted in the mobile base in cartesian space."""
+        odometry = self.odometry
```

Comment:
This maybe will work better 
<code> 
def translate_by(self, x: float, y: float, timeout: Optional[float] = None) -> None:
    odometry = self.odometry
    x_current = odometry["x"]
    y_current = odometry["y"]
    theta = odometry["theta"]
    theta_rad = deg2rad(theta)
    x_goal = x_current + (x * cos(theta_rad) - y * sin(theta_rad))
    y_goal = y_current + (x * sin(theta_rad) + y * cos(theta_rad))
    self.goto(x_goal, y_goal, theta, timeout=timeout)
</code>